frontend-web/webclient/app/Accounting/Allocations.tsx

```diff
@@ -648,7 +689,9 @@ function stateReducer(state: State, action: UIAction): State {
                     unit: string,
                     usage: number,
                     quota: number,
-                    maxUsable: number
+                    maxUsable: number,
+                    retired: number,
```

Comment:
Might make more sense to call it "retiredAmount" as it otherwise sounds like a boolean value